pkg/ansibleapp/types.go

```diff
@@ -2,32 +2,45 @@ package ansibleapp
 
 import (
 	"encoding/json"
+	"github.com/pborman/uuid"
 )
 
+type Parameters map[string]interface{}
 type SpecManifest map[string]*Spec
 
+type AnswerDescriptor struct {
+	Name        string      `json:"name"`
+	Description string      `json:"description"`
+	Default     interface{} `json:"default"`
```

Comment:
It could be a string, I debated this briefly when writing this and chose `interface{}` because of things like ports. This will serialize correctly into a json blob also, so it will show `{"default": 400}` instead of `{"default": "400"}`.

I can update if we don't see much of a benefit here, it just feels more correct?